### PR TITLE
Add wikidata link

### PR DESCRIPTION
**Pull request by bekicot**
Wikidata is a collaboratively edited knowledge base operated by the
Wikimedia Foundation.

Kind of surprise, one of the largest collaboratively edited database that is _**always**_ free to use is not available here :D

### CODE REVIEW

**Comment by jbrooksuk** [comment]
@davemachado @mikestreety what do we think about the new section?

**Comment by bekicot** [comment]
@jbrooksuk Sory for creating new section. Wikidata is just "Fit" into almost all of the orher api provided. It is a knowledge base, and it should have data for any knowledge Human have. :)

**Comment by mikestreety** [comment]
To me, it seems like it belongs in the [data-acess](https://github.com/toddmotto/public-apis#data-access) section?

**Comment by bekicot** [comment]
@mikestreety 
Yes could be. In my opinion, by adding a new section will broaden the view of this kind of database. For example https://en.wikipedia.org/wiki/Knowledge_Graph .

**Comment by jbrooksuk** [comment]
We do try to refrain from adding more sections at this point. It might be that we go through them and see what we can split a little or rename, but that’ll be in the future I imagine.

**Comment by davemachado** [comment]
I stand by my opinion from earlier PR's - keep sections concise until they are too varied, then split.

**Comment by bekicot** [comment]
Alright, changed into data access section :)

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Please note a passing build status indicates all listed APIs are available since
 * [Games & Comics](#games--comics)
 * [Geocoding](#geocoding)
 * [Health](#health)
+* [Knowledge Base](#knowledge-base)
```
Comment (davemachado) [inline README.md]: What is this new section referring to?

Comment (davemachado) [inline README.md]: What is this new section referring to?

Comment (bekicot) [inline README.md]: @davemachado Whoops, fixed now :)

**Comment by davemachado** [comment]
Thanks!

**Pull request merged**